hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OzoneAclUtil.java

```diff
@@ -46,9 +45,6 @@
 public final class OzoneAclUtil {
   static final Logger LOG = LoggerFactory.getLogger(OzoneAclUtil.class);
 
-  private static ACLType[] userRights;
```

Comment:
I would prefer keep these two global variables, to avoid creating the short lived collections object and array during each create key request.  Others look good to me.

Reply:
Thanks @ChenSammi for the review.  Collections object is not short-lived, because `OzoneManager` keeps a single instance of `OmConfig`, which uses the `MemoizedSupplier` to remember the `List<ACLType` collection.  Only the result of `toArray()` is short-lived.  I'll update the PR to avoid that.